CRITICAL - 2022-11-22 20:52:49 --> syntax error, unexpected '}', expecting ';'
in APPPATH\Controllers\AuthAdmin.php on line 29.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\AuthAdmin.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthAdmin')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthAdmin')
 4 [internal function]: spl_autoload_call('App\\Controllers\\AuthAdmin')
 5 SYSTEMPATH\CodeIgniter.php(877): class_exists('\\App\\Controllers\\AuthAdmin', true)
 6 SYSTEMPATH\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-11-22 20:57:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'krs', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\ci4-krs\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\ci4-krs\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ci4\ci4-krs\app\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
#8 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
#9 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\ci4-krs\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\ci4-krs\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ci4\ci4-krs\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4\ci4-krs\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\ci4\ci4-krs\app\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
#7 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
#8 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4\ci4-krs\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4\ci4-krs\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-22 20:57:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email_user` = :email_user:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-11-22 20:57:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'krs', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\ci4-krs\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\ci4-krs\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ci4\ci4-krs\app\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
#8 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
#9 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\ci4-krs\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\ci4-krs\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\ci4\ci4-krs\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\ci4-krs\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ci4\ci4-krs\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4\ci4-krs\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\ci4\ci4-krs\app\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
#7 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
#8 C:\xampp\htdocs\ci4\ci4-krs\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4\ci4-krs\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4\ci4-krs\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-22 20:57:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email_user` = :email_user:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthAdmin.php(25): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-11-22 21:23:33 --> syntax error, unexpected 'endif' (T_ENDIF), expecting end of file
in APPPATH\Views\auth\loginadmin.php on line 70.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('auth/loginadmin', [], true)
 3 APPPATH\Controllers\AuthAdmin.php(17): view('auth/loginadmin')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->login()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-11-22 21:23:48 --> syntax error, unexpected 'endif' (T_ENDIF), expecting end of file
in APPPATH\Views\auth\loginadmin.php on line 70.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('auth/loginadmin', [], true)
 3 APPPATH\Controllers\AuthAdmin.php(17): view('auth/loginadmin')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->login()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-11-22 21:24:20 --> Trying to get property 'true' of non-object
in APPPATH\Controllers\AuthAdmin.php on line 33.
 1 APPPATH\Controllers\AuthAdmin.php(33): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'true\' of non-object', 'APPPATH\\Controllers\\AuthAdmin.php', 33, [...])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-11-22 21:26:56 --> Trying to get property 'true' of non-object
in APPPATH\Controllers\AuthAdmin.php on line 33.
 1 APPPATH\Controllers\AuthAdmin.php(33): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'true\' of non-object', 'APPPATH\\Controllers\\AuthAdmin.php', 33, [...])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-11-22 21:33:56 --> Undefined variable: checkPassword
in APPPATH\Controllers\AuthAdmin.php on line 32.
 1 APPPATH\Controllers\AuthAdmin.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: checkPassword', 'APPPATH\\Controllers\\AuthAdmin.php', 32, [...])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\AuthAdmin->prosesLogin()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthAdmin))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
